browser/src/view/components/Content/ContentCollection.scss

```diff
@@ -0,0 +1,55 @@
+@use '../../../styles/common/variables' as *;
+
+table {
+  font-family: Arial, Helvetica, sans-serif;
+  width: 100%;
+  height: 100%;
+  border-collapse: collapse;
+  border: 1px solid $COLOR-TABLE-BORDER;
+  border-radius: 0.5rem;
+  box-shadow: 0px 4px 8px -2px rgba(16, 24, 40, 0.1), 0px 2px 4px -2px rgba(16, 24, 40, 0.06);
+}
+
+th {
+  height: 2.75rem;
+  background-color: $COLOR-TABLE-HEADER-BG;
+  text-align: left;
+  margin: 0;
+  padding-left: 1.5rem;
+}
+
+tr {
+  height: 4.5rem;
+  border-bottom: 1px solid $COLOR-TABLE-BORDER;
+}
+
+td {
+  padding: 0 1.5rem;
+}
+
+.skeleton {
+  width: 100%;
+  height: 1rem;
+  margin-bottom: 0.25rem;
+  border-radius: 1rem;
+  background-color: darkgrey;
+  animation: skeleton-loading 1.5s linear infinite alternate;
+}
```

Comment:
Noting that we should probably use a mixin here given that we'll most likely want to make height, width, border-radius, etc configurable.